docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+version: '3.7'
+services:
+  snapper:
+    build: .
+    ports:
+      - '8000:8000'
+    volumes:
+      - snapper_output:/snapper_output
+    command: --output_paths_format "http://localhost:8080{}"
```

Comment:
I'd like to pass the FQDN value as an environment variable. The "localhost" value might a default one.

Reply:
SNAPPER_FQDN env variable. Default is http://localhost

Reply:
https://github.com/revisor48/Snapper/pull/3/commits/9643092a81ab47d9e3e8028da31eb00475bae92b